### PR TITLE
add access-logs doc

### PR DESCRIPTION
**Pull request by KannarFr**

### CODE REVIEW

File: commons/tools/access-logs.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+title: Access Logs
+shortdesc: Get and process your access logs.
+tags:
+- apps, metrics, accesslogs, warp10
+---
+
+# Access Logs analysis
+
+All your applications access logs are now pushed to Warp10. You are now able to process them directly on the console on the Metrics tab of your applications.
```
Comment (urcadox) [inline commons/tools/access-logs.md]: C'est pas un blog post, donc pas de "are now", juste "now". Il faut que ce soit timeless.

Comment (urcadox) [reply commons/tools/access-logs.md]: s/on/in/ aussi

File: commons/tools/access-logs.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+tS -> Haproxy termination_state
+adc -> Reverse proxy hostname
+w -> workerId (Sozu)
+r -> requestId (Sozu)
+```
+
+# A Warp10 query example:
+
+```bash
+# retrieve all accesslogs from now to last 30s
```
Comment (urcadox) [inline commons/tools/access-logs.md]: s/from to last 30s/since 30 seconds ago/

File: commons/tools/access-logs.md (hunk 1 above)
Comment (KannarFr) [reply commons/tools/access-logs.md]: Les deux on to in?

**Comment by KannarFr** [comment]
@vballu Hi, on your last commit, the script creates a new GTS with the status codes, right? It needs a write token? Which is not provided to our users. To fetch status code classically, you can use the script I gave in example but there is no GTS created,  so I think this example can be removed.

WDYT?

File: commons/tools/access-logs.md (hunk 1 above)
Comment (urcadox) [reply commons/tools/access-logs.md]: oui

**Comment by vballu** [comment]
@KannarFr 

Exactly, the example script create a GTS as output with the status codes as values. yet, they have no write operations. This is just a convenient format

**Comment by KannarFr** [comment]
@vballu As users can't write, I'm not sure this example is interesting :/.

**Comment by vballu** [comment]
@KannarFr, GTS is a format ! It is a convenient way to handle data. Even us, we often use this kind of format without any write operations.

This example, show how to use `accessLogs` data and integrate it in an other warp10 treatement. Users can use the output GTS for lot of others usages, like filtering by values which is far easier if it's a GTS.

**Comment by KannarFr** [comment]
Ok then, can you add this explanation on the document?

**Comment by vballu** [comment]
@KannarFr , I add some explanations. You right, it's better with it

**Comment by KannarFr** [comment]
I think we need to go further and explain what is a GTS, why it's convenient, WDYT?

**Comment by vballu** [comment]
So, we need a section about our metrics and technologie behind. That is much than only examples

File: commons/tools/warp10.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+Warp10 is a time series database. The notion of `class`, `labels`, `longitude`, `latitude`, `altitude` and `value` are used.
+
+A GTS is a GeoTime Serie define by a `class` and somes `labels`. They are indexed and used to quickly retrived the data.
+
+`labels` is a kind of dictionnary. That is called a **map** under the warp10 terminology
+
+A GTS may contain some values which have the following models : `[ timestamp longitude latitude altitude value ]`
+
+The `warpscript` langage is the langage run by warp10. It is a **reverse polish notation** with the notion of **stack**
```
Comment (ldoguin) [inline commons/tools/warp10.md]: Warp10 uses `warpscript`. It's a stack based language using **reverse polish notation** .

File: commons/tools/warp10.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+> The metrics ovh team build an other way to request warp10: https://github.com/ovh/tsl/
```
Comment (ldoguin) [inline commons/tools/warp10.md]: built*

File: commons/tools/warp10.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+> Notice that the timestamp is in **milliseconds**
+
+> [Warp1O documentation is availlable in their website](https://www.warp10.io/doc/reference)
```
Comment (ldoguin) [inline commons/tools/warp10.md]: on their website

File: commons/tools/warp10.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+## Endpoint
+
+The clevercloud warp10 endpoint is:
```
Comment (ldoguin) [inline commons/tools/warp10.md]: Clever Cloud

File: commons/tools/warp10.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+```
+https://c1-warp10-clevercloud-customers.services.clever-cloud.com/api/v0
+```
+
+You can find some documentation about endpoint gateway [here](https://www.warp10.io/content/03_Documentation/03_Interacting_with_Warp_10/01_Introduction).
```
Comment (ldoguin) [inline commons/tools/warp10.md]: You can find documentation about endpoint gateway [here](https://www.warp10.io/content/03_Documentation/03_Interacting_with_Warp_10/01_Introduction).

File: commons/tools/warp10.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+
+> You can find the endpoint and an available token under the `metric` tab of your application
+
+Hence, you can request our warp10 platform by your own script. With a curl for expample :
```
Comment (ldoguin) [inline commons/tools/warp10.md]: You can query our warp10 platform with your own script. Here's a curl example :

File: commons/tools/warp10.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+> Do not forget the endpoint. `exec` in the previous exemple.
+
+## Token
+
+Tokens are based on your application with the notion of producer and owner. Hence, only the data owner can see it.
+
+You can find a 5 days available token in the `metric` tab of your application.
+
+## technicals constraints
```
Comment (ldoguin) [inline commons/tools/warp10.md]: Technical

File: commons/tools/warp10.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+| [MAXBUCKETS](https://www.warp10.io/doc/MAXBUCKETS) | maximum number of buckets which can be created by a call to BUCKETIZE | 10e5 | 50e5 |
+| [MAXDEPTH](https://www.warp10.io/doc/MAXDEPTH) | maximum depth (number of levels) of the execution stack | 5e3 | 5e3 |
+| [MAXLOOP](https://www.warp10.io/doc/MAXLOOP) | maximum number of **milliseconds** which can be spent in a loop | 5e3 | 10e3 |
+| [MAXOPS](https://www.warp10.io/doc/MAXOPS) | maximum number of operations which can be performed during a single WarpScript execution | 5e6 | 5e7 |
+| [MAXSYMBOLS](https://www.warp10.io/doc/MAXSYMBOLS) | maximum number of simultaneous symbols which can be defined on the stack during a single WarpScript execution | 64 | 256 |
+| [MAXGEOCELLS](https://www.warp10.io/doc/MAXGEOCELLS) | maximum number of cells a GEOSHAPE  | 10e3 | 10e4 |
+| [MAXPIXELS](https://www.warp10.io/doc/MAXPIXELS) | maximum size (in pixels) of images which can be created by PGraphics | 10e5 | 10e5 |
+| [MAXRECURSION](https://www.warp10.io/doc/MAXRECURSION) | maximum nesting depth of macro calls | 16 | 32 |
+
+> **NOTICE THAT OPERATIONS OVER SOFT LIMITS MAY BE REALLY  INTENSIVES AND SLOWS. THEY SHOULDN'T NOT BE USED**
```
Comment (ldoguin) [inline commons/tools/warp10.md]: REALLY INTENSIVES

THEY SHOULD NOT BE USED

File: commons/tools/warp10.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+
+## Token
+
+Tokens are based on your application with the notion of producer and owner. Hence, only the data owner can see it.
+
+You can find a 5 days available token in the `metric` tab of your application.
+
+## technicals constraints
+
+We had fixed some limits in our Warp10 database. For each we have a *soft* and a *hard* one. To pass over the *soft* on. The stack must be [`AUTHENTICATE`](https://www.warp10.io/doc/AUTHENTICATE) and the limit must be **explicitly** set under the maximum define by the *hard*.
```
Comment (ldoguin) [inline commons/tools/warp10.md]: Too convoluted, please rewrite it.

File: commons/tools/warp10.md [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+```warpscript
+'<READTOKEN>' AUTHENTICATE
+50e6 TOLONG LIMIT
+// Fetch on the 'accessLogs' class for your application id as labels
+[ '<READTOKEN>' 'accessLogs' { 'app_id' '<APP_ID>'  } NOW 1 w ] FETCH
+```
+
+# Quantum
+
+Quantum is a web tool use to run some warpscript. You can access to it from your metrics interface.
```
Comment (ldoguin) [inline commons/tools/warp10.md]: used*

File: commons/tools/warp10.md [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+
+It provide the path to the clevercloud warp10 gateway and let you explore your data.
```
Comment (ldoguin) [inline commons/tools/warp10.md]: Clever Cloud

**Comment by vballu** [comment]
go to pull request #425

**Pull request closed without merging**